Cargo.toml

```diff
@@ -3,18 +3,18 @@ resolver = "2"
 members = ["build", "contracts", "ffi", "steel"]
 
 [workspace.package]
-version = "0.11.0-alpha.1"
+version = "1.1.0-alpha.1"
 edition = "2021"
 license = "Apache-2.0"
 homepage = "https://risczero.com/"
 repository = "https://github.com/risc0/risc0-ethereum/"
 
 [workspace.dependencies]
 # Intra-workspace dependencies
-risc0-build-ethereum = { version = "0.11.0-alpha.1", default-features = false, path = "build" }
-risc0-ethereum-contracts = { version = "0.11.0-alpha.1", default-features = false, path = "contracts" }
-risc0-steel = { version = "0.11.0-alpha.1", default-features = false, path = "steel" }
-risc0-forge-ffi = { version = "0.11.0-alpha.1", default-features = false, path = "ffi" }
+risc0-build-ethereum = { version = "1.1.0-alpha.1", default-features = false, path = "build" }
+risc0-ethereum-contracts = { version = "0.12.0-alpha.1", default-features = false, path = "contracts" }
+risc0-steel = { version = "1.1.0-alpha.1", default-features = false, path = "steel" }
```

Comment:
```suggestion
risc0-ethereum-contracts = { version = "1.1.0-alpha.1", default-features = false, path = "contracts" }
risc0-steel = { version = "0.12.0-alpha.1", default-features = false, path = "steel" }
```

Reply:
Thank you! I fixed this with https://github.com/risc0/risc0-ethereum/pull/139/commits/77b046add8a1cdf8c362ace90e87ddd8c4379a87